src/utils/version.ts

```diff
@@ -154,13 +151,9 @@ export function checkForUpdates(): void {
     return;
   }
 
-  const message = `Update available ${chalk.dim(
-    notifier.update.current
-  )}${chalk.reset(' → ')}${chalk.green(
-    notifier.update.latest
-  )}\nRun ${chalk.cyan('yarn global add ')}${chalk.cyan(pkg.name)} to update`;
-
-  notifier.notify({ defer: false, message });
+  notifier.notify({
+    defer: false,
+  });
```

Comment:
What is the default message, now that we're no longer passing one?

Reply:
This is what the default message looks like:
![https://raw.githubusercontent.com/yeoman/update-notifier/main/screenshot.png](update notification)

We added the customization to suggest `yarn` when needed but this was done upstream now: yeoman/update-notifier#132